python/mcap/mcap/exceptions.py

```diff
@@ -13,3 +18,10 @@ class DecoderNotFoundError(McapError):
 
 class EndOfFile(McapError):
     pass
+
+
+class InvalidRecordLength(McapError):
+    def __init__(self, opcode: Opcode, length: int, limit: int):
+        super().__init__(
+            f"{opcode.name} record has invalid length {length}, limit is set to {limit}",
+        )
```

Comment:
Is the record length actually invalid, or just too big?

```suggestion
class RecordLengthLimitExceeded(McapError):
    def __init__(self, opcode: Opcode, length: int, limit: int):
        super().__init__(
            f"{opcode.name} record has length {length}, limit is set to {limit}",
        )
```